lib/thor/util.rb

```diff
@@ -211,7 +211,7 @@ def thor_root_glob
       #
       def globs_for(path)
         path = escape_globs(path)
-        ["#{path}/Thorfile", "#{path}/*.thor", "#{path}/tasks/*.thor", "#{path}/lib/tasks/*.thor"]
+        ["#{path}/Thorfile", "#{path}/*.thor", "#{path}/tasks/*.thor", "#{path}/lib/tasks/*.thor", "#{path}/lib/tasks/**/*.thor"]
```

Comment:
Does this also work?
```suggestion
        ["#{path}/Thorfile", "#{path}/*.thor", "#{path}/tasks/*.thor", "#{path}/lib/tasks/**/*.thor"]
```

Reply:
@rafaelfranca 
Thank you for the review!
Yes, that works too! My code was redundant, I fix it.

Reply:
@rafaelfranca 
Fixed.
I have confirmed that the specs all passed.
Please review again 😌